### PR TITLE
ReleaseNotes: Updated changelog and release notes for 8.0.1

### PR DESCRIPTION
**Pull request by grafanabot**
This exciting! So much has changed!

### CODE REVIEW

**Review by osg-grafana — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,27 @@
...
+
+* **Alerting/SSE:** Fix "count_non_null" reducer validation. [#35451](https://github.com/grafana/grafana/pull/35451), [@kylebrandt](https://github.com/kylebrandt)
+* **Cloudwatch:** Fix duplicated time series. [#35433](https://github.com/grafana/grafana/pull/35433), [@sunker](https://github.com/sunker)
+* **Cloudwatch:** Fix missing defaultRegion. [#35436](https://github.com/grafana/grafana/pull/35436), [@andresmgot](https://github.com/andresmgot)
+* **Dashboard:** Fix Dashboard init failed error on dashboards with old singlestat panels in collapsed rows. [#35425](https://github.com/grafana/grafana/pull/35425), [@torkelo](https://github.com/torkelo)
+* **Datasource:** Fix storing timeout option as numeric. [#35441](https://github.com/grafana/grafana/pull/35441), [@marefr](https://github.com/marefr)
+* **Postgres/MySQL/MSSQL:** Fix annotation parsing for empty responses. [#35367](https://github.com/grafana/grafana/pull/35367), [@marcbachmann](https://github.com/marcbachmann)
+* **Postgres/MySQL/MSSQL:** Fix so that numeric/non-string values are returned from query variables. [#35411](https://github.com/grafana/grafana/pull/35411), [@marefr](https://github.com/marefr)
```
Comment (achatterjee-grafana) [inline CHANGELOG.md:14]:
```suggestion
* **Postgres/MySQL/MSSQL:** Numeric/non-string values are now returned from query variables. [#35411](https://github.com/grafana/grafana/pull/35411), [@marefr](https://github.com/marefr)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,27 @@
...
+* **Postgres:** Fix error thrown when annotation query does not return any results. [#35382](https://github.com/grafana/grafana/pull/35382), [@dprokop](https://github.com/dprokop)
```
Comment (achatterjee-grafana) [inline CHANGELOG.md:15]:
```suggestion
* **Postgres:** Fix an error that was thrown when the annotation query did not return any results. [#35382](https://github.com/grafana/grafana/pull/35382), [@dprokop](https://github.com/dprokop)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,27 @@
...
+* **StatPanel:** Fix issue with graph look when switching color mode. [#35460](https://github.com/grafana/grafana/pull/35460), [@torkelo](https://github.com/torkelo)
```
Comment (achatterjee-grafana) [inline CHANGELOG.md:16]:
```suggestion
* **StatPanel:** Fix an issue with the appearance of the graph when switching color mode. [#35460](https://github.com/grafana/grafana/pull/35460), [@torkelo](https://github.com/torkelo)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,27 @@
...
+* **Visualizations:** Fix Stat/BarGauge/Gauge/PieChart all values mode showing same name if same value. [#35368](https://github.com/grafana/grafana/pull/35368), [@torkelo](https://github.com/torkelo)
```
Comment (achatterjee-grafana) [inline CHANGELOG.md:17]:
```suggestion
* **Visualizations:** Fix an issue in the Stat/BarGauge/Gauge/PieChart panels where all values mode were showing the same name if they had the same value. [#35368](https://github.com/grafana/grafana/pull/35368), [@torkelo](https://github.com/torkelo)
```

File: docs/sources/release-notes/release-notes-8-0-1.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+* **Postgres/MySQL/MSSQL:** Fix annotation parsing for empty responses. [#35367](https://github.com/grafana/grafana/pull/35367), [@marcbachmann](https://github.com/marcbachmann)
+* **Postgres/MySQL/MSSQL:** Fix so that numeric/non-string values are returned from query variables. [#35411](https://github.com/grafana/grafana/pull/35411), [@marefr](https://github.com/marefr)
+* **Postgres:** Fix error thrown when annotation query does not return any results. [#35382](https://github.com/grafana/grafana/pull/35382), [@dprokop](https://github.com/dprokop)
+* **StatPanel:** Fix issue with graph look when switching color mode. [#35460](https://github.com/grafana/grafana/pull/35460), [@torkelo](https://github.com/torkelo)
+* **Visualizations:** Fix Stat/BarGauge/Gauge/PieChart all values mode showing same name if same value. [#35368](https://github.com/grafana/grafana/pull/35368), [@torkelo](https://github.com/torkelo)
+
+### Plugin development fixes & changes
+
+* **Toolkit:** Resolve external fonts when Grafana is served from a sub path. [#35352](https://github.com/grafana/grafana/pull/35352), [@jackw](https://github.com/jackw)
+
```
Comment (achatterjee-grafana) [inline docs/sources/release-notes/release-notes-8-0-1.md:27]: Same comments as above.

**Review by achatterjee-grafana — APPROVED** [review]

**Pull request merged**